.github/workflows/velox_be.yml

```diff
@@ -21,6 +21,7 @@ on:
       - '.github/**'
       - 'pom.xml'
       - 'backends-velox/**'
+      - 'gluten-celeborn/**'
```

Comment:
I suppose changing celeborn's code should trigger the velox tests too. Are it OK to include this change in this PR or should make this change separately?